rfc/12/README.md

```diff
@@ -81,4 +81,4 @@ There are 2 options for health checks integration into Pacemaker:
 
 ## See also
 
-* [Original list of LDR1 HA health checks. The spreadsheet has a lot of useful information](https://docs.google.com/spreadsheets/d/1xASlPnlFx1OmhKttbgOweHbmfbXMI6n3Pmp2TSO8ou8/edit#gid=1305517710)
+* [Original list of LDR1 HA health checks. The spreadsheet has a lot of useful information](https://seagatetechnology.sharepoint.com/:x:/r/sites/gteamdrv1/tdrive1224/LDR%20R1/HA/HA%20requirements%202020-01-14%20.xlsx?d=wdf76a255f0b84af5a771744ef00440d5&csf=1&web=1&e=gKXtZM)
```

Comment:
If the sharing path that is used currently is from its original location (which may not be in Hare project directory), then document can get vanished if somebody deletes it from original location.
Were the documents moved to sharepoint's Hare directory and then sharing link is created?

Reply:
This is not a Hare document but HA. We must not move other component documents in Hare directory. If documents are vanishing from the folders, then this needs to be reported to IT and to all the teams. May be such URLs can be removed from the document if not required. But this can figured out later. Let's just fix the links in this PR's scope.